setup.py

```diff
@@ -74,18 +74,16 @@
             ],
             # following are AiiDA plugin entry points:
             'aiida.calculations': [
-                'calculation = aiida.orm.calculation:Calculation',
-                'function = aiida.orm.calculation.function:FunctionCalculation',
-                'job = aiida.orm.calculation.job:JobCalculation',
-                'work = aiida.orm.calculation.work:WorkCalculation',
-                'simpleplugins.arithmetic.add = aiida.orm.calculation.job.simpleplugins.arithmetic.add:ArithmeticAddCalculation',
-                'simpleplugins.templatereplacer = aiida.orm.calculation.job.simpleplugins.templatereplacer:TemplatereplacerCalculation',
+                'arithmetic.add = aiida.calculations.plugins.arithmetic.add:ArithmeticAddCalculation',
+                'templatereplacer = aiida.calculations.plugins.templatereplacer:TemplatereplacerCalculation',
```

Comment:
Should we add entries for arithmetic and templatereplacer in the plugin registry, to reserve the names?

Reply:
Yes, if you agree with the renaming, we should do this. I did not like the `simpleplugins` prefix, but if we could find a good namespace to put these internal ones in that'd be ok too. If we think it is not a problem to reserve these namespace, then we should register them on the registry